### PR TITLE
[MCC-1114446] Fixing unit tests and adding uuid attribute to Carmen::Country

### PR DESCRIPTION
**Pull request by johnzhangmdsol**

### CODE REVIEW

File: lib/carmen/region.rb [hunk 1]
```diff
@@ -90,7 +90,7 @@ def load_subregions_from_path(path, parent=nil)
     # and overlaying matching data (if it exists) from the overlay_path.
     def load_data_at_path(path)
       data_sets = Carmen.data_paths.map do |data_path|
-        if File.exists?(data_path + path)
+        if File.exist?(data_path + path)
```
Comment (edkim-mdsol) [inline lib/carmen/region.rb:93]: can we use `File.join` here?

File: spec/carmen/sanity_spec.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@
   end
 
   it "has 248 countries" do
-    Carmen::Country.all.size.must_equal 248
+    Carmen::Country.all.size.must_equal 249
```
Comment (edkim-mdsol) [inline spec/carmen/sanity_spec.rb:15]: if there's a new country, can we change line 14 to also reflect it?

File: lib/carmen/region.rb (hunk 1 above)
Comment (johnzhangmdsol) [reply lib/carmen/region.rb:93]: We'll end up calling .exist? anyways once we get the file join - join just creates a path :)

**Review by edkim-mdsol — APPROVED** [review]

**Comment by johnzhangmdsol** [comment]
Reverted this PR by pushing up previous version of master. We will not make a merge into master until everything is completed.

**Pull request merged**